eq-author-api/knexfile.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   },
   acquireConnectionTimeout: 5000,
   migrations: {
-    tableName: "knex_migrations"
+    tableName: "knex_migrations",
+    directory: "../migrations"
```

Comment:
Now that the config lives in the root of the project, shouldn't this be `./migrations` or omitted?